spec/views/hq/partners/index_spec.rb

```diff
@@ -1,11 +1,27 @@
 require 'rails_helper'
+require 'will_paginate/array'
 
 RSpec.describe "hq/partners/index.html.erb", type: :view do
   context 'partners exist' do
+    let(:partners) do
+      (1..31).each_with_object([]) do |num, arr|
+        arr << FactoryGirl.build_stubbed(:partner)
+      end.paginate(:page => 2)
```

Comment:
@Nik, will_paginate has a default per_page of 30 items. As you know, I tried 5 per_page and it looked too sparse. Since we have 12 partners, is there a specific per_page that you'd like?